Description: Avoid hardcoding mocha reporter or colorization
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2019-10-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -31,7 +31,7 @@
 NODE ?= $(shell which node)
 NPM ?= $(shell which npm)
 UGLIFYJS ?= ./node_modules/.bin/uglifyjs
-MOCHA ?= ./node_modules/.bin/mocha
+MOCHA ?= ./node_modules/.bin/mocha -R dot -C
 MUSTACHE ?= ./node_modules/.bin/mustache
 ESLINT ?= ./node_modules/.bin/eslint
 JSDOC ?= ./node_modules/.bin/jsdoc
@@ -66,10 +66,10 @@
 benchmark: perf/*_perf.js
 
 test: node_modules lunr.js
-	${MOCHA} test/*.js -u tdd -r test/test_helper.js -R dot -C
+	${MOCHA} test/*.js -u tdd -r test/test_helper.js
 
 test/inspect: node_modules lunr.js
-	${MOCHA} test/*.js -u tdd -r test/test_helper.js -R dot -C --inspect-brk=0.0.0.0:9292
+	${MOCHA} test/*.js -u tdd -r test/test_helper.js --inspect-brk=0.0.0.0:9292
 
 test/env/file_list.json: $(wildcard test/*test.js)
 	${NODE} -p 'JSON.stringify({test_files: process.argv.slice(1)})' $^ > $@
